.github/workflows/compare_translations.py

```diff
@@ -115,5 +119,18 @@ def check_translations():
 
 
 if __name__ == "__main__":
-    check_translations()
+    parser = argparse.ArgumentParser(
+        description="Check and print missing translations for all non-default languages."
+    )
+    parser.add_argument(
+        "--directory", type=str, help="Directory containing translation files."
```

Comment:
1. Add "relative to the `root` directory"
1. Use the `default` flag to set a default value
    1.  https://docs.python.org/3/library/argparse.html#default
1. Test that the directory exists, and fail if it doesn't using a non-zero exit code